### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by Giliaz**

### CODE REVIEW

File: home_flask_1.py [hunk 1]
```diff
@@ -14,16 +14,22 @@ def index():
...
-    while True:
-        # создаем рандомній список 10-20 символов
-        password = ([chr(random.randrange(33, 127)) for _ in range(random.randrange(10, 21))])
-        # конвертируем список в строку
-        pas_check = ''.join(password)
-        # проверяым регулярным выражением на соответствие пароля ТЗ
-        if re.match(r'(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[!"#$%&(){}\'*+,-./\\:;<>=?@\[\]^_`|~])', pas_check):
-            break
+    password = ([chr(random.randrange(33, 127)) for _ in range(random.randrange(10, 21))])
+    pas_check = ''.join(password)
```
Comment (mikolaz27) [inline home_flask_1.py]: should be password_check not pas_check

File: home_flask_1.py [hunk 2]
```diff
@@ -14,16 +14,22 @@ def index():
...
 def generate_password():
-    while True:
-        # создаем рандомній список 10-20 символов
-        password = ([chr(random.randrange(33, 127)) for _ in range(random.randrange(10, 21))])
-        # конвертируем список в строку
-        pas_check = ''.join(password)
-        # проверяым регулярным выражением на соответствие пароля ТЗ
-        if re.match(r'(?=.*[0-9])(?=.*[a-z])(?=.*[A-Z])(?=.*[!"#$%&(){}\'*+,-./\\:;<>=?@\[\]^_`|~])', pas_check):
-            break
+    password = ([chr(random.randrange(33, 127)) for _ in range(random.randrange(10, 21))])
```
Comment (mikolaz27) [inline home_flask_1.py]: why do you use randrange here? And why do you have this numbers 33 and 127?

File: home_flask_1.py [hunk 3]
```diff
@@ -14,16 +14,22 @@ def index():
...
+    pas_check = ''.join(password)
     return f"<center><h2><u>Password lenght {len(password)} symb.</u>:  {pas_check}</h2></center>"
 
+#  єто я так задолбался в первій раз)))
```
Comment (mikolaz27) [inline home_flask_1.py]: remove commented code

File: home_flask_1.py [hunk 4]
```diff
@@ -14,16 +14,22 @@ def index():
 #passwoerd
```
Comment (mikolaz27) [inline home_flask_1.py:14]: remove commented code

**Review by mikolaz27 — CHANGES_REQUESTED** [review]
use meaningful name of branches not just new_branch

File: home_flask_1.py (hunk 2 above)
Comment (Giliaz) [reply home_flask_1.py]: "And why do you have this numbers 33 and 127?....." This ASCII code password symbols

File: home_flask_1.py (hunk 1 above)
Comment (Giliaz) [reply home_flask_1.py]: corrected...

File: home_flask_1.py (hunk 3 above)
Comment (Giliaz) [reply home_flask_1.py]: corrected..

File: home_flask_1.py (hunk 4 above)
Comment (Giliaz) [reply home_flask_1.py:14]: corrected...

File: home_flask_1.py (hunk 2 above)
Comment (Giliaz) [reply home_flask_1.py]: This line generates from 10 to 20 codes ASCII, and translates them into a list of characters... Still, comments are sometimes help..)))

> why do you use randrange here? And why do you have this numbers 33 and 127?

**Comment by Giliaz** [comment]
> use meaningful name of branches not just new_branch
ok, I heard you

**Pull request closed without merging**